doc/developer/lua.rst

```diff
@@ -0,0 +1,59 @@
+LUA
+===
+
+LUA is currently experimental within FRR and has very limited
+support.  If you would like to compile FRR with LUA you must
+follow these steps:
+
+1. Installation of Relevant Libraries
+
+   .. code-block:: console
+       apt-get install lua5.3 liblua5-3 liblua5.3-dev
+
+   These are the Debian libraries that are needed.  There should
+   be equivalent RPM's that can be found
+
+2. Compilation
+
+   Configure needs these options
+
+   .. code-block:: console
+       ./configure --enable-dev-build --enable-lua <all other interesting options>
```

Comment:
3 space indent, and a blank line after the `code-block` line

Reply:
fixed